upstairs/src/downstairs.rs

```diff
@@ -819,12 +819,12 @@ impl Downstairs {
             return true;
         }
         // If this client is offline, then move it to faulted after skipping
-        // all the outstanding jobs.  This will shut the door on this client till
-        // the deactivation has finished on all downstairs and the upstairs has
-        // completed deactivation.  After skipping jobs and setting faulted,
-        // we return false here and let the faulting framework take care of
-        // clearing out the skipped jobs.  This then allows the requested
-        // deactivation to finish.
+        // all the outstanding jobs.  This will shut the door on this client
```

Comment:
Fixing this one line that was too long resulted in changing the entire comment.
No actual difference here, just moving words around.